node/src/chain_spec.rs

```diff
@@ -33,32 +35,6 @@ pub type DevelopmentChainSpec =
 /// The default XCM version to set in genesis config.
 const SAFE_XCM_VERSION: u32 = xcm::prelude::XCM_VERSION;
 
-const AMPLITUDE_PARACHAIN_ID: u32 = 2124;
-const AMPLITUDE_INITIAL_ISSUANCE: Balance = 200_000_000 * UNIT;
-const INITIAL_ISSUANCE_PER_SIGNATORY: Balance = 200 * UNIT;
-const INITIAL_COLLATOR_STAKING: Balance = 10_010 * UNIT;
-
-const TOKEN_DECIMALS: u32 = 12;
-
-const INITIAL_AMPLITUDE_SUDO_SIGNATORIES: [&str; 5] = [
-	"6nJwMD3gk36fe6pMRL2UpbwAEjDdjjxdngQGShe753pyAvCT",
-	"6i4xDEE1Q2Bv8tnJtgD4jse4YTAzzCwCJVUehRQ93hCqKp8f",
-	"6n62KZWvmZHgeyEXTvQFmoHRMqjKfFWvQVsApkePekuNfek5",
-	"6kwxQBRKMadrY9Lq3K8gZXkw1UkjacpqYhcqvX3AqmN9DofF",
-	"6kKHwcpCVC18KepwvdMSME8Q7ZTNr1RoRUrFDH9AdAmhL3Pt",
-];
-
-const INITIAL_AMPLITUDE_VALIDATORS: [&str; 8] = [
-	"6mTATq7Ug9RPk4s8aMv5H7WVZ7RvwrJ1JitbYMXWPhanzqiv",
-	"6n8WiWqjEB8nCNRo5mxXc89FqhuMd2dgXNSrzuPxoZSnatnL",
-	"6ic56zZmjqo746yifWzcNxxzxLe3pRo8WNitotniUQvgKnyU",
-	"6gvFApEyYj4EavJP26mwbVu7YxFBYZ9gaJFB7gv5gA6vNfze",
-	"6mz3ymVAsfHotEhHphVRvLLBhMZ2frnwbuvW5QZiMRwJghxE",
-	"6mpD3zcHcUBkxCjTsGg2tMTfmQZdXLVYZnk4UkN2XAUTLkRe",
-	"6mGcZntk59RK2JfxfdmprgDJeByVUgaffMQYkp1ZeoEKeBJA",
-	"6jq7obxC7AxhWeJNzopwYidKNNe48cLrbGSgB2zs2SuRTWGA",
-];
-
 // For mainnet USDC issued by centre.io
 const MAINNET_DEFAULT_WRAPPED_CURRENCY_ID: CurrencyId = CurrencyId::Stellar(Asset::AlphaNum4 {
```

Comment:
Should we also move this to the `constants/amplitude.rs` file maybe?

Reply:
Or no, let's remove this constant altogether. It's the same as the `MAINNET_USDC_CURRENCY_ID` that is defined in the constants. So let's just remove this constant here and use refer to the other constant in the places where this is used.

Reply:
Amazing. Done!